### PR TITLE
fix: Fix paremeter send to validate function

### PR DESCRIPTION
**Pull request by RaphaelOliveiraMoura**
I changed the paremeter send to validate function. The function expect as firts parameter the obj res of express, but it was receiving req object.

PS: Another suggestion is export isnomia or postman api documentation for another developers test your api.

And, congratulations for the work, It was amazing. 👏🏻

### CODE REVIEW

File: src/app/validators/User.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ class UserValidator {
     });
 
     // Validating the input data
```
Comment (LauraBeatris) [inline src/app/validators/User.js:17]: Can you remove that comment too? I don't remember why I wrote that, it's kind of unnecessary if you know the meaning/semantic of the function  🤦‍♀️

**Comment by LauraBeatris** [comment]
@RaphaelOliveiraMoura Thanks for the contribution 💝

Indeed the ```validate``` utility function expects a ```request``` instead of a ```response``` since it needs to access the input data. Good catch! 

I just added one thing during the review, that was my mistake actually 🤦‍♀️

Looking forward to continuing with the discussion.

File: src/app/validators/User.js (hunk 1 above)
Comment (RaphaelOliveiraMoura) [reply src/app/validators/User.js]: done 😉

**Comment by LauraBeatris** [comment]
Merged! Thanks :smile:

**Pull request merged**